go/vt/vtorc/inst/instance_dao.go

```diff
@@ -1210,3 +1210,20 @@ func ExpireStaleInstanceBinlogCoordinates() error {
 	}
 	return ExecDBWriteFunc(writeFunc)
 }
+
+// SnapshotDatabaseState takes the snapshot of the database and returns it.
+func SnapshotDatabaseState() (string, error) {
+	var finalOutput string
+	for _, tableName := range db.TableNames {
+		finalOutput += "Table " + tableName + "\n"
```

Comment:
We probably don't hit this too often, but using strings.Builder would be more efficient. IMO this would also be much nicer if it returned valid JSON. That makes it much easier to integrate with other tools/systems (e.g. as a human with `jq` and other systems that might poll and alert).

Reply:
Good idea, I made it json now 👍